mysql-db/seed.js

```diff
@@ -2,14 +2,24 @@ const mysql = require('mysql');
 const faker = require('faker');
 const Promise = require('bluebird');
 
-const connection = mysql.createConnection({
+// const connection = mysql.createConnection({
+//   host: 'localhost',
+//   user: 'root',
+//   password: '',
```

Comment:
Do you want to create a .gitignored file on your local and use variables for your DB username and password so it doesn't end up on GitHub/deployed? I don't know if it matters for mySQL but may be a good habit to get into for any kind of log-in/password situation?

Reply:
Good point! The CSV files make this script unnecessary, but you're right that it's good practice so I'll include it